gm4_crossbow_cartridges/data/gm4_crossbow_cartridges/functions/projectile/torch.mcfunction

```diff
@@ -1,18 +1,22 @@
 # Shoot torch
 # @s = player using the crossbow
-# at @s
+# at arrow
 # run from check_projectile
```

Comment:
Should add that this can also be called from itself, same in other files in this folder.